server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -118,9 +118,12 @@ function bigNumber(big, suffixes, base) {
  * Formats the timestamp nicely
  */
 function dateFormat(timestamp) {
-   var date = new Date(timestamp);
-   date.toLocaleString().split(' ').join('&nbsp;');
-   return date;
+   var d = new Date(timestamp);
+   var day = d.getDate();
+   var month = d.getMonth() + 1;
+   var year = d.getFullYear();
+   var dateStr = "" + year + "-" + month + "-" + day + " " + d.getHours() + ":" + d.getMinutes() + ":" + d.getSeconds() + "," + d.getMilliseconds();
+   return dateStr;
```

Comment:
This formatting is one very specific format, but the actual log files are fully configurable. Trying to match this to whatever users might have in their log file format isn't very useful. We'll match one preference, sure, but only by diverging from another. The locale-specific string is better, I think. If we want to make this configurable, we can add a configuration option later to allow users to tweak the format.

Reply:
Yeah I agree.  I can pull these changes out into another request.  But the problem I was seeing was that I didn't notice any difference in the dates display between just `new Date()` or ` date.toLocaleString()` or `date.toLocaleDateString()`.  It could have just been Chrome though since that was the only browser I tried it in. I also didn't want to have to import a new js library just to format one date but if we want anything else we might have to.  We can discuss more in another PR.